### PR TITLE
infix to postfix.c

### PR DESCRIPTION
**Pull request by parizadas**

### CODE REVIEW

File: Task 3/infix-to-postfix.c [hunk 1]
```diff
@@ -0,0 +1,114 @@
+#include<stdio.h>
```
Comment (nimishbongale) [inline Task 3/infix-to-postfix.c:1]: Yes! Better. Now just one final thing. 

You'll have to put this in your own folder.

So your full path should be something like:

`Task 3/InfixToPostfix/infix-to-postfix.c`

Can you add a multiline comment on the top of the program to show what the program does?

**Review by nimishbongale — COMMENTED** [review]
Nearly there!

One small change. Let's get this merged!

File: Task 3/infix-to-postfix.c (hunk 1 above)
Comment (nimishbongale) [reply Task 3/infix-to-postfix.c:1]: You can also add the output screenshot in your own folder! 😄

Comment (parizadas) [reply Task 3/infix-to-postfix.c:1]: how to make a folder...please guide me

Comment (nimishbongale) [reply Task 3/infix-to-postfix.c:2]: If you are committing via the web app, you have to enter a slash ('/') in the filename part.

But fyi, this isn't a recommended way. You need to use something known as a `git client`. Find it here:

-  https://git-scm.com/

**Review by nimishbongale — APPROVED** [review]
@parizadas 

Good work 🚀 

I know it's your first time contributing, and so I went ahead & created the folder for you. 

But I hope this has been a learning experience for you. 😄 

Please make sure you install the git client & learn some basic git commands before contributing elsewhere! Good luck!

**Comment by parizadas** [comment]
@nimishbongale Thankyou so much for your help.... Yes it was a learning experience... sure I will explore more on git. Once again thankyou.

**Pull request merged**